components/contact-form/index.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC } from "react";
+
+export interface ContactFormProps {}
+
+const ContactForm: FC<ContactFormProps> = () => {
+  return (
+    <div className="ContactForm__Container">
```

Comment:
Lets drop the __Container, name this 'ContactForm' name other things that are parts of it with our __etc. convention.